src/pages/fileBrowser/fileBrowser.include.js

```diff
@@ -204,6 +205,14 @@ function FileBrowserInclude(mode, info, doesOpenLast = true) {
         toast(strings.success);
         return;
       }
+
+      if(action === 'copy-path'){

```

Comment:
Actually there is no need for this feature because this will copy absolute path and user can't use it anywhere. I'll update the app to add paths in autocomplete like vscode does very soon.

Reply:
Oo , that's nice

I had added it because many times, user is not able to get the path of dir